server/routes.go

```diff
@@ -42,8 +44,4 @@ func (srv *AppServer) setupRoutesAndMiddleware() {
 	srv.router.Use(middleware.AddSecurityHeaders(config.EnableTls))
 	srv.router.Use(middleware.HandleFaviconRequests())
 
-	if config.EnableDetailedRequestLogging {
```

Comment:
Are you effectively removing this check? Is that intended?

Reply:
yes. I intended to make the logs detailed for both valid and invalid endpoint, so removed the check